### DISCUSSION TITLE
Find all pods with linkerd sidecars injected in them

### DISCUSSION THREAD

**Discussion by drewwells** [linkerd/linkerd2#5147] [General] [answered]
### What is the issue?
Linkerd does not use labels for sidecar injection, so it's not possible to sort for pods that have an injected sidecar.

### How can it be reproduced?
`kubectl get po -A --annotation linkerd.io/inject=enabled` This command does not exist, only labels are filterable. https://github.com/kubernetes/kubernetes/issues/18801#issuecomment-530025114

### Possible solution
Migrate to labels

### Additional context
#5022 has discussion on other reasons that annotations should not be used for sidecar decisions

**Comment by grampelberg** [comment, marked as answer]
Here's how you'd do that:

```
kubectl get deploy \
    -o custom-columns=NAME:.metadata.name,"ANNOTATIONS":".spec.template.metadata.annotations.linkerd\.io/inject"  \
    --no-headers | \
grep enabled
```

Note: labels would be great, this is just an immediate solution.
